### ISSUE TITLE
A Mindmap to organize the content

### ISSUE THREAD

**Issue by ZandorZ** [mtdvio/every-programmer-should-know#149]
I think it'd be awesome if we had a big mindmap to organize all the topics and subtopics.

**Comment by siriniok** [comment]
Do you have any examples to consider? Are there any tools for generating mindmaps?

**Comment by shotgunner** [comment]
@siriniok take a look at [learn-anything](https://learn-anything.xyz/) and its [repo](https://github.com/learn-anything/learn-anything/tree/master/app)

**Comment by siriniok** [comment]
@shotgunner yeah, thanks, was thinking about it. Need to take a detailed look.
